CONTRIBUTING.md

```diff
@@ -79,19 +79,22 @@ about Git.
 The title of the pull request should be prefixed by the component or area that
 the pull request affects. Valid areas as:
 
-  - `consensus` for changes to consensus critical code
```

Comment:
why is this (re)moved?

Reply:
Figured they should be ordered by simplicity.